src/main/webapp/WEB-INF/view/notifications.jsp

```diff
@@ -28,14 +28,24 @@
       %>
       <%if(type.equals("Conversation")){%>
         <li> <strong> <%= Date.from(notification.getCreationTime()) %>: </strong>
-            <%= Username %> opened a conversation!
-            Conversation title <%= notifcationInfo.get(1)%>
+          <div class="alert alert-info alert-dismissible fade show" role="alert">
+            <strong><%= Username %></strong> opened a conversation!
+            Conversation:   <a href="/chat/<%= notifcationInfo.get(2) %>"> <%= notifcationInfo.get(1)%> </a>
+            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
```

Comment:
You can remove this button, since it'll just hide the alert but not the date title above it.
Alternatively, move the the date from line 30 above the username here, and make it a `div` so they display on its own line. This way clicking the x hides the entire row.